fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -53671,10 +53671,10 @@ is_a: FBbt:00007116 ! presumptive embryonic/larval system
 [Term]
 id: FBbt:00005710
 name: embryonic fat body
-def: "Fat body of the embryo." [FBC:SPR]
+def: "Fat body of the embryo. This structure is a single-cell thick, lateral ribbon with dorsal fingers extending from the posterior region. Holes within the ribbon are formed by the intrusion of various organs (Hoshizaki et al., 1994)." [FBC:CP, FBC:SPR, FlyBase:FBrf0073371]
```

Comment:
Add stage specificity here.  Based on def: 'fat body primordium' applies before Stage 16 ("the embryonic fat body is morphologically identifiable at stage 16 (Hoshizaki et al., 1994)") => embryonic fat body applies during S16 & 17.  Suspect you'll find lots of cases in FB where 'embryonic fat body' is used for earlier stages.  Probably worth flagging to Sian.